remoteit/desktop
# [DESK-676, DESK-674, DESK-675, DESK-660, DESK-452] AWS AMI support

### Description


Show platform icon
if AWS platform
1. Hide network scanning
2. Raise target services limit to 100

### Test plan



1. Sign in to an AWS instance device
2. See platform icons in device list
3. Do not see the network scan button
4. Be able to add 100 services (or at least more than 10)

### Ticket



<https://remoteit.atlassian.net/browse/DESK-676>
<https://remoteit.atlassian.net/browse/DESK-674>
<https://remoteit.atlassian.net/browse/DESK-675>
<https://remoteit.atlassian.net/browse/DESK-660>
<https://remoteit.atlassian.net/browse/DESK-452>
